kratos/includes/dof.h

```diff
@@ -579,13 +471,19 @@ class Dof : public IndexedObject
 
 
     /** True is is fixed */
-    bool mIsFixed;
+    int mIsFixed : 1;
+
+    int mVariableType : 3;
+
+    int mReactionType : 3;
+
+    int mIndex : 8;
 
     /** Equation identificator of the degree of freedom */
-    EquationIdType mEquationId;
+    EquationIdType mEquationId : 48;
```

Comment:
could you briefly explain what those numbers mean?
I think they are related to my above suggestions